src/domain/synthetics/trade/utils/swapStats.ts

```diff
@@ -19,16 +19,41 @@ export function getSwapPathOutputAddresses(p: {
   swapPath: string[];
   wrappedNativeTokenAddress: string;
   shouldUnwrapNativeToken: boolean;
+  isIncrease: boolean;
 }) {
-  const { marketsInfoData, initialCollateralAddress, swapPath, wrappedNativeTokenAddress, shouldUnwrapNativeToken } = p;
+  const {
+    marketsInfoData,
+    initialCollateralAddress,
+    swapPath,
+    wrappedNativeTokenAddress,
+    shouldUnwrapNativeToken,
+    isIncrease,
+  } = p;
 
   if (swapPath.length === 0) {
-    return {
-      outTokenAddress:
-        shouldUnwrapNativeToken && initialCollateralAddress === wrappedNativeTokenAddress
-          ? NATIVE_TOKEN_ADDRESS
-          : initialCollateralAddress,
+    // If the swap path is empty and is increase order,
+    // we do not care about the output token address during happy path,
+    // as user will not be getting any token in return.
```

Comment:
this sounds misleading. user is not getting any token right now cos he's paying, not selling. But it doesn't mean he won't get it back

Reply:
-||-
rephrased